src/main/java/com/fasterxml/jackson/module/jsonSchema/types/ArraySchema.java

```diff
@@ -189,6 +211,24 @@ public boolean equals(Object obj) {
 		@Override
 		public boolean isArrayItems() { return true; }
 	}
+
+	public static class ItemsDeserializer extends JsonDeserializer<Items> {
+
+		@Override
+		public Items deserialize(JsonParser parser,
+								 DeserializationContext context) throws IOException, JsonProcessingException {
+			ObjectCodec mapper = parser.getCodec();
```

Comment:
I assume it is legal to use array, or a single item here?
This works, I can probably simplify it after merge a bit.